ServicesDelete.php

```
@@ -0,0 +1,237 @@
+    }
+
+    // Dummy password check, replace with real logic (e.g., check against env or DB)
+    private function checkPassword($password): bool
+    {
+        // Example: check against an environment variable
+        return $password === env('ADMIN_PASSWORD', 'your_default_password');
+    }
+
+    private function deleteServer()
+    {
+        $servers = Server::all();
```

PHPMD (2.15.0)

Avoid using static access to class '\App\Models\Server' in method 'deleteServer'. (undefined)

(StaticAccess)